src/skin/legacy/tooltips.cpp

```diff
@@ -391,13 +391,23 @@ void Tooltips::addStandardTooltips() {
             << tr("BPM Tap")
             << tr("When tapped repeatedly, adjusts the BPM to match the tapped BPM.");
 
+    add("beats_set_marker")
+            << tr("Set Beat Marker")
+            << tr("Set a beat marker at the current play position.");
+
+    add("beats_remove_marker")
+            << tr("Remove Beat Marker")
+            << tr("Remove the beat marker at the current play position.");
+
     add("beats_adjust_slower")
             << tr("Adjust BPM Down")
-            << tr("When tapped, adjusts the average BPM down by a small amount.");
+            << tr("When tapped, decrease the BPM of the region around the "
+                  "current play position by a small amount.");
```

Comment:
My expectation would be: 
```suggestion
                  "current play position by a small amount and shift the following regions accordingly.");
```
is this correct?

Reply:
No, it just increases/decreases the `beatsTillNextMarker` value by a whole bar, but the marker positions stay the same.

Reply:
Yes, I have written the comments before I have discovered that.

We need to make a usability check if this works out.

When would one use this feature? 
I original had something like this in mind, but I am afraid it is not (yet) possible:

1.) Set the first bar 
2.) Move in the region after the bar, adjust the tempo.
3.) Play the track and verify the beats 
4.) Identify a bar with an offset 
5.) Adjust the bar by implicit setting a marker.
6.) Continue at 2.

Later you may recognize that there is a beat or bar rollover in the a region. 
In case we assume that the following marker is set correct. You current approach works. 

In case the user has messed up the beat grids and likes to start over from a certain position, the gentle tempo adjusting inclusive shifting all later beats might be desired.